com.brave.Brave.yaml

```diff
@@ -0,0 +1,81 @@
+app-id: com.brave.Brave
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: brave
+separate-locales: false
+build-options:
+  no-debuginfo: true
+finish-args:
+  - '--device=all'
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=pulseaudio'
+  - '--socket=x11'
+  - '--socket=wayland'
+  - '--socket=cups'
+  - '--talk-name=org.freedesktop.FileManager1'
+  - '--talk-name=org.freedesktop.Notifications'
+  - '--talk-name=org.freedesktop.secrets'
+  - '--talk-name=org.freedesktop.UPower'
+  - '--talk-name=org.freedesktop.GeoClue2'
+  - '--talk-name=org.gnome.SessionManager'
+  - '--own-name=org.mpris.MediaPlayer2.*'
+  - '--filesystem=host-etc'
+  - '--filesystem=xdg-documents'
+  - '--filesystem=xdg-download'
+  - '--filesystem=xdg-music'
+  - '--filesystem=xdg-videos'
+modules:
+  - name: brave
+    buildsystem: simple
+    build-commands:
+      - 'install -Dm 755 brave.sh /app/bin/brave'
+      - 'install -Dm 755 apply_extra.sh /app/bin/apply_extra'
+      - 'install -Dm 755 stub_sandbox.sh /app/bin/stub_sandbox'
+      - 'install -Dm 644 -t /app/share/applications com.brave.Brave.desktop'
+      - 'install -Dm 644 -t /app/share/metainfo com.brave.Brave.metainfo.xml'
+      - |
+          for icon in 16 24 32 48 64 128 256; do
+            install -Dm 644 com.brave.Brave-$icon.png /app/share/icons/hicolor/${icon}x${icon}/apps/com.brave.Brave.png
+          done
+
+      # For extra-data to be able to extract the .deb
+      - 'install -Dm 755 /{usr,app}/bin/ar'
+      - 'cp /usr/lib/$(gcc --print-multiarch)/libbfd-*.so /app/lib'
+    sources:
+      - type: extra-data
```

Comment:
By the way, if Brave's license supports this, I think you can get rid of the `extra-data` business and use `file`. It's way more simple...but I don't know about the license restrictions, etc.

Reply:
@x80486 Can you talk more about what this means and what sort of license restrictions you are unsure about?

Reply:
You would have to wait for Flathub's owners to explain this better, but I think it boils down to whether Brave allows or not to redistribute/repackage the browser...is that right, @barthalion?

Reply:
analotia raised a good point about widevine so I guess extra-data needs to stay.